scripts/helper_functions.sh

```diff
@@ -2,6 +2,14 @@
 
 # Common functions definitions
 
+function wait_for_apt
+{
+    while fuser /var/lib/dpkg/lock-frontend >/dev/null 2>&1 ; do
+        echo "Waiting for front-end lock"
```

Comment:
Can we save this in a log file say

echo  " `date`  Waiting for front-end lock " > /tmp/lock-frontend.log